scripts/compress_file.py

```diff
@@ -0,0 +1,26 @@
+#!python3
+
+import sys
+
+import init_scripts  # pylint: disable=unused-import
+
+from app.helpers.utils import gzip_string
+
+
+def main():
+    if len(sys.argv) < 2:
+        print('File argument missing')
+        sys.exit(-1)
+
+    file = sys.argv[1]
+    with open(file, mode='r', encoding='utf-8') as fd:
+        data = fd.read()
+
+    gz_data = gzip_string(data)
+
+    with open(file + '.gz', 'wb') as fd:
+        fd.write(gz_data)
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
maybe it would be good to use the standard `gzip` to create the compressed file, otherwise we have the risk that if sth is wrong with our `gzip_string` we produce invalid test-data

Reply:
@boecklic The issue is that the standard unix `gzip` program doesn't uses the same gzip header as the python module and then in the test I cannot check the file length. Note that for the moment, when uploading files that are gzipped, they are unzipped to check if the content is a valid kml file, and then gzipped again using the python module.
You are right that using here the `gzip_string()` is a bit dangerous but this script has been used once to create the gzipped samples and is not used anymore more. I put it here as documentation and to a add future gzipped sample file if needed, but this script is not used during unit tests.